Makefile

```diff
@@ -639,7 +638,6 @@ SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
             drivers/rx_nrf24l01.c \
             drivers/rx_spi.c \
             drivers/rx_xn297.c \
-            drivers/pwm_output.c \
```

Comment:
@mikeller : Can you link the statement?

One reason may be that mixing different optimization levels may be quite stressing on the compiler, it may uncover unexpected bugs (with LTO, optimization on file and function levels are almost identical)

@martinbudden : In case of CF/BF, the portability should not me that bit problem. Other parts of (low level) code are GCC specific anyway (some of them are my fault [ or "design decision"]).
IMO it will be sufficient to use one global optimization level when building for targets other than microprocessor - other targets have no memory/speed constraints. So other compiler just needs to ignore function level optimizations